installer/dev/WindowsAppRuntimeInstall.vcxproj

```diff
@@ -42,7 +42,8 @@
   </ItemGroup>
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
```

Comment:
Why?

Are you using VS2022? v143 smells like it. Most everything uses v143 (and no conditional logic)

Reply:
Yes, I am using VS2022. v143 is to accommodate that. Most of WindowsAppRuntime dll projects all use v142 and some are using the conditional logic. I went with conditional logic just to be like that some! But yes, I am fine with just v143.

Reply:
No, most don't

* # referencing v142 = 65
* # referencing v143 = 2 (RestartAgent and Microsoft.Windows.ApplicationModel.Resources) and both conditionally

@kythant is working on the plan to convert us to building with VS2022 but today we (still) require VS2019 to build WinAppSDK

Reply:
Ok, I am also saying the same thing that v142 is used most currently. But I learnt now that you and Will are co-ordinating to swtich to using VS 2022 and v143 soon. So I will remove the change to v143 altogether even with conditional usage of v142 and let you both do the switch for all projects at once. Thanks.